include/grpc/support/dynamic_annotations.h

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
As mentioned in the other PR, please move this file to src/core/lib/iomgr.

Reply:
Sure, moved to iomgr.